Add touch to create the logfile if it does not exist

oyucekaya opened the pull request:
The file is now created when it does not exists instead of causing an blocking error. I think this is a better solution because, if you wanted to migrate to another server you will get a blocking error if you forget to create the file.

Peardian commented:
It looks good to me. However, this change causes the related test to fail (see Travis results), and I don't want to merge anything that damages the stability. The test simply checks for the exception that is now prevented. Would you mind removing that test? A proper test to check the new behavior can be added later.

oyucekaya commented:
The tests are fixed. We made new function called testSetLogPathThrowsException|() tot test whether the code throws an exception when a false filename is given. We changed the testSetLogPath() function and added an extra unklink in the tearDown function, because the created file during a testrun has to be deleted after the tests are completed.

Peardian commented:
Thanks, that's better. It looks good, so I will merge it when I am able to.

Peardian commented:
Oh no, I didn't realize it was targed to the wrong branch...
